cpp/src/c_api/graph_sg.cpp

```diff
@@ -398,6 +420,22 @@ struct create_graph_csr_functor : public cugraph::c_api::abstract_functor {
         cugraph::graph_view_t<vertex_t, edge_t, store_transposed, multi_gpu>,
         edge_type_id_t>(handle_);
 
+      if (symmetrize_) {
+        if (edgelist_edge_ids || edgelist_edge_types) {
```

Comment:
We should add an issue to discuss what symmetrization means with respect to edge properties.

Reply:
I tracked this in an [issue](https://github.com/rapidsai/cugraph/issues/4675)